app/routes/API_Demo/Demo/Demo.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import { EmptyLayout } from "./../../../components";
+
+const Demo = () => {
+const [page, setPage] = useState(0)
+const [stories, setStories] = useState([]);
+  
+  useEffect(() => {
+    fetchData();
+  },[]);
+
+  async function fetchData() {
+     
+    let  url = `http://localhost:3001/stories?page=${page}`
```

Comment:
This port number shouldn't be hard-coded (it's determined dynamically, eg my default is 4001)

Reply:
Fixed 👍